src/ports/ngenuity/templates/README.md

```diff
@@ -0,0 +1,48 @@
+{{{ basic-heading }}}
+
+# Installation
+
+1. Download one of the available profiles and extract it
+2. Click on `Presets` in the upper right corner
+3. Click on `Add` and select de extracted file
+
+{{{ done }}}
+
+# Profiles
+
+### [João Pedro Alves](https://github.com/joaopealves)
+
+![img]()
+
+## [Download]()
```

Comment:
Download link is missing, update it to:

`https://github.com/daltonmenezes/aura-theme/blob/main/packages/ngnuity/profiles/joaopealves/Aura%20Theme.hxp.zip?raw=true`